src/ApiService/ApiService/UserCredentials.cs

```diff
@@ -51,12 +51,12 @@ public class UserCredentials {
     }
 
 
-    static async Task<OneFuzzResult<string[]>> GetAllowedTenants() {
-        return OneFuzzResult<string[]>.Ok(Array.Empty<string>());
+    static Task<OneFuzzResult<string[]>> GetAllowedTenants() {
```

Comment:
this is not a proper implementation, it's just a stub. I'm waiting for Instance Config model to be available

Reply:
I just changed it  because it was causing a warning since you have an async function with no aait